serving/docs/adapters_api.md

```diff
@@ -29,6 +31,22 @@ curl -X POST "http://localhost:8080/models/adaptecho/adapters?name=a1&src=..."
 }
 ```
 
+### Update an adapter
+
+`POST /models/{model_name}/adapters/{adapter_name}/update`
+
+* src - The adapter src. It currently requires a file, but eventually an id or URL can be supported depending on the model handler.
+* pin (optional): Whether to pin the adapter.
```

Comment:
can we add a short description of what pinning means? if it's already in the docs elsewhere that's good too.